src/assets/stylesheets/base_styles/Colors.md

```diff
@@ -0,0 +1,39 @@
+Our brand colors:
+
+
+    const styles = require('assets/stylesheets/base_styles/colors_map.scss');
+
+    function stylesArray() {
+      let all_colors = [[],[],[],[]];
+      let types = ['neutral', 'brand', 'accent', 'action'];
+
+      for (style in styles) {
+        types.forEach( (type, idx) => {
+          if( styles[style].includes(type) ) {
```

Comment:
not sure you need to look up the `style` here; shouldn't the `style` itself include the `type` string?